gran_source_list.tsv

```diff
@@ -11,7 +11,11 @@ usgs-r/EflowStats	v4.0.2
 GLEON/GLMr	v3.1.1
 ldecicco-usgs/EGRETci	v0.0.8
 usgs-r/smwrBase	1.0.4
-usgs-r/smwrGraphs	1.0.7
+usgs-r/smwrGraphs	v1.0.10
 usgs-r/glmtools	v0.9.3
 usgs-r/powstreams	v0.5.1
+<<<<<<< HEAD
```

Comment:
OK, it did not make sense to me. What needs to be done?


On Mon, May 18, 2015 at 3:06 PM, Jordan Walker <notifications@github.com>
wrote:

> In gran_source_list.tsv
> <https://github.com/USGS-R/grantools/pull/50#discussion_r30542213>:
>
> >  usgs-r/glmtools	v0.9.3
> >  usgs-r/powstreams	v0.5.1
> > +<<<<<<< HEAD
>
> Looks like you checked in the merge conflict. This should be cleaned up
> before merging.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/USGS-R/grantools/pull/50/files#r30542213>.
>